CRITICAL - 2023-06-05 01:55:39 --> Invalid file: "Login/listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Login/listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('Login/listado', [], true)
 3 APPPATH\Controllers\Login.php(14): view('Login/listado')
 4 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 01:56:16 --> Invalid file: "Login/listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Login/listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('Login/listado', [], true)
 3 APPPATH\Controllers\Login.php(14): view('Login/listado')
 4 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 02:02:12 --> Invalid file: "Login/listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Login/listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('Login/listado', [], true)
 3 APPPATH\Controllers\Login.php(14): view('Login/listado')
 4 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 02:32:39 --> Class "App\Models\CrudModel" not found
in APPPATH\Controllers\Crud.php on line 9.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 02:34:58 --> mysqli_sql_exception: Table 'sistema_db.t_personas' doesn't exist in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT * FROM t...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM t...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM t...')
#3 C:\xampp\htdocs\codeigniter4\app\Models\CrudModel.php(7): CodeIgniter\Database\BaseConnection->query('SELECT * FROM t...')
#4 C:\xampp\htdocs\codeigniter4\app\Controllers\Crud.php(10): App\Models\CrudModel->listarNombres()
#5 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(934): App\Controllers\Crud->listado()
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\codeigniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-06-05 02:34:58 --> Table 'sistema_db.t_personas' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT * FROM t_personas', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM t_personas')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM t_personas')
 4 APPPATH\Models\CrudModel.php(7): CodeIgniter\Database\BaseConnection->query('SELECT * FROM t_personas')
 5 APPPATH\Controllers\Crud.php(10): App\Models\CrudModel->listarNombres()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 02:38:13 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 02:39:44 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 02:41:27 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 02:59:45 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 03:10:32 --> mysqli_sql_exception: Incorrect table definition; there can be only one auto column and it must be defined as a key in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\codeigniter4\app\Database\Migrations\2023-06-05-030550_Users.php(45): CodeIgniter\Database\Forge->createTable('usuers')
#5 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(877): App\Database\Migrations\Users->up()
#6 C:\xampp\htdocs\codeigniter4\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\codeigniter4\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\codeigniter4\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\codeigniter4\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\codeigniter4\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}
CRITICAL - 2023-06-05 03:12:05 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:23:57 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:27:03 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:29:49 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:29:51 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:29:52 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:29:53 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:30:05 --> Invalid file: "listado.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('listado.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 3 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:30:18 --> Undefined property: stdClass::$nombre
in APPPATH\Views\listado.php on line 57.
 1 APPPATH\Views\listado.php(57): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$nombre', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\listado.php', 57)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\listado.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 5 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:30:24 --> Undefined property: stdClass::$nombre
in APPPATH\Views\listado.php on line 57.
 1 APPPATH\Views\listado.php(57): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$nombre', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\listado.php', 57)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\listado.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 5 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:32:36 --> Undefined property: stdClass::$nombre
in APPPATH\Views\listado.php on line 58.
 1 APPPATH\Views\listado.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$nombre', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\listado.php', 58)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\listado.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 5 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:32:38 --> Undefined property: stdClass::$nombre
in APPPATH\Views\listado.php on line 58.
 1 APPPATH\Views\listado.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$nombre', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\listado.php', 58)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\listado.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 5 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 03:33:10 --> Undefined property: stdClass::$contraseña
in APPPATH\Views\listado.php on line 61.
 1 APPPATH\Views\listado.php(61): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$contraseña', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\listado.php', 61)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\listado.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('listado', [], true)
 5 APPPATH\Controllers\Crud.php(19): view('listado', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->listado()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 04:19:56 --> syntax error, unexpected token "}"
in APPPATH\Controllers\Crud.php on line 25.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Crud')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Crud')
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\Crud', true)
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 04:59:16 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 04:59:46 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', ' ', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:00:24 --> Undefined array key "contraseña"
in APPPATH\Controllers\Crud.php on line 27.
 1 APPPATH\Controllers\Crud.php(27): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "contraseña"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 27)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:03:40 --> Undefined array key "contraseña"
in APPPATH\Controllers\Crud.php on line 27.
 1 APPPATH\Controllers\Crud.php(27): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "contraseña"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 27)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 05:04:23 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\app\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codeigniter4\app\Controllers\Crud.php(32): App\Models\CrudModel->insertar(Array)
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(934): App\Controllers\Crud->crear()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-06-05 05:04:23 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `telefono`, `direccion`) VALUES (\'6\', \'6\', \'6\', \'6\', \'6\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`nombre`, `apellido`, `email`, `telefono`, `direccion`) VALUES (\'6\', \'6\', \'6\', \'6\', \'6\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`nombre`, `apellido`, `email`, `telefono`, `direccion`) VALUES (\'6\', \'6\', \'6\', \'6\', \'6\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `telefono`, `direccion`) VALUES (:nombre:, :apellido:, :email:, :telefono:, :direccion:)', [...], false)
 5 APPPATH\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Crud.php(32): App\Models\CrudModel->insertar([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:05:07 --> Undefined array key "name"
in APPPATH\Controllers\Crud.php on line 24.
 1 APPPATH\Controllers\Crud.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "name"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 24)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:05:43 --> Undefined array key "name"
in APPPATH\Controllers\Crud.php on line 24.
 1 APPPATH\Controllers\Crud.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "name"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 24)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:05:53 --> Undefined array key "name"
in APPPATH\Controllers\Crud.php on line 24.
 1 APPPATH\Controllers\Crud.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "name"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 24)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:06:16 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:06:38 --> Undefined array key "name"
in APPPATH\Controllers\Crud.php on line 24.
 1 APPPATH\Controllers\Crud.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "name"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 24)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:06:38 --> Undefined array key "name"
in APPPATH\Controllers\Crud.php on line 24.
 1 APPPATH\Controllers\Crud.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "name"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 24)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:07:24 --> Undefined array key "contraseña"
in APPPATH\Controllers\Crud.php on line 27.
 1 APPPATH\Controllers\Crud.php(27): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "contraseña"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 27)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:07:39 --> Undefined array key "contraseña"
in APPPATH\Controllers\Crud.php on line 27.
 1 APPPATH\Controllers\Crud.php(27): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "contraseña"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 27)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 05:07:59 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\app\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codeigniter4\app\Controllers\Crud.php(32): App\Models\CrudModel->insertar(Array)
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(934): App\Controllers\Crud->crear()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-06-05 05:07:59 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (:nombre:, :apellido:, :email:, :clave:, :telefono:, :direccion:)', [...], false)
 5 APPPATH\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Crud.php(32): App\Models\CrudModel->insertar([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:08:32 --> Undefined array key "name"
in APPPATH\Controllers\Crud.php on line 24.
 1 APPPATH\Controllers\Crud.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "name"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 24)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 05:09:45 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\app\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codeigniter4\app\Controllers\Crud.php(32): App\Models\CrudModel->insertar(Array)
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(934): App\Controllers\Crud->crear()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-06-05 05:09:45 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (:nombre:, :apellido:, :email:, :clave:, :telefono:, :direccion:)', [...], false)
 5 APPPATH\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Crud.php(32): App\Models\CrudModel->insertar([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 05:10:26 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\app\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codeigniter4\app\Controllers\Crud.php(32): App\Models\CrudModel->insertar(Array)
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(934): App\Controllers\Crud->crear()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-06-05 05:10:26 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (:nombre:, :apellido:, :email:, :clave:, :telefono:, :direccion:)', [...], false)
 5 APPPATH\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Crud.php(32): App\Models\CrudModel->insertar([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 05:10:59 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\app\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codeigniter4\app\Controllers\Crud.php(32): App\Models\CrudModel->insertar(Array)
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(934): App\Controllers\Crud->crear()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-06-05 05:10:59 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (:nombre:, :apellido:, :email:, :clave:, :telefono:, :direccion:)', [...], false)
 5 APPPATH\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Crud.php(32): App\Models\CrudModel->insertar([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 05:11:17 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\app\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codeigniter4\app\Controllers\Crud.php(32): App\Models\CrudModel->insertar(Array)
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(934): App\Controllers\Crud->crear()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-06-05 05:11:17 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (:nombre:, :apellido:, :email:, :clave:, :telefono:, :direccion:)', [...], false)
 5 APPPATH\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Crud.php(32): App\Models\CrudModel->insertar([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:11:42 --> Undefined array key "name"
in APPPATH\Controllers\Crud.php on line 24.
 1 APPPATH\Controllers\Crud.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "name"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 24)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 05:11:56 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\app\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codeigniter4\app\Controllers\Crud.php(32): App\Models\CrudModel->insertar(Array)
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(934): App\Controllers\Crud->crear()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-06-05 05:11:56 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (:nombre:, :apellido:, :email:, :clave:, :telefono:, :direccion:)', [...], false)
 5 APPPATH\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Crud.php(32): App\Models\CrudModel->insertar([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 05:14:57 --> mysqli_sql_exception: Unknown column 'nombre' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\app\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codeigniter4\app\Controllers\Crud.php(32): App\Models\CrudModel->insertar(Array)
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(934): App\Controllers\Crud->crear()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-06-05 05:14:57 --> Unknown column 'nombre' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`nombre`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (:nombre:, :apellido:, :email:, :clave:, :telefono:, :direccion:)', [...], false)
 5 APPPATH\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Crud.php(32): App\Models\CrudModel->insertar([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:16:00 --> Undefined array key "name"
in APPPATH\Controllers\Crud.php on line 24.
 1 APPPATH\Controllers\Crud.php(24): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "name"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 24)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-05 05:16:20 --> mysqli_sql_exception: Unknown column 'clave' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\app\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\codeigniter4\app\Controllers\Crud.php(32): App\Models\CrudModel->insertar(Array)
#6 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(934): App\Controllers\Crud->crear()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-06-05 05:16:20 --> Unknown column 'clave' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `users` (`name`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`name`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`name`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (\'8\', \'8\', \'8\', \'8\', \'8\', \'8\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`name`, `apellido`, `email`, `clave`, `telefono`, `direccion`) VALUES (:name:, :apellido:, :email:, :clave:, :telefono:, :direccion:)', [...], false)
 5 APPPATH\Models\CrudModel.php(12): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Crud.php(32): App\Models\CrudModel->insertar([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->crear()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:16:41 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:17:09 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:19:15 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:25:33 --> Undefined array key "id_nombre"
in APPPATH\Views\actualizar.php on line 4.
 1 APPPATH\Views\actualizar.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_nombre"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\actualizar.php', 4)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\actualizar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('actualizar', [], true)
 5 APPPATH\Controllers\Crud.php(70): view('actualizar', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->obtenerNombre('5')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:26:03 --> Undefined array key "id_nombre"
in APPPATH\Views\actualizar.php on line 4.
 1 APPPATH\Views\actualizar.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_nombre"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\actualizar.php', 4)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\actualizar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('actualizar', [], true)
 5 APPPATH\Controllers\Crud.php(70): view('actualizar', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->obtenerNombre('5')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:29:46 --> Undefined array key "id"
in APPPATH\Controllers\Crud.php on line 50.
 1 APPPATH\Controllers\Crud.php(50): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 50)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->actualizar()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:31:25 --> Undefined array key "id"
in APPPATH\Controllers\Crud.php on line 50.
 1 APPPATH\Controllers\Crud.php(50): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 50)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->actualizar()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:33:42 --> Undefined array key "id"
in APPPATH\Controllers\Crud.php on line 50.
 1 APPPATH\Controllers\Crud.php(50): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 50)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->actualizar()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:35:22 --> Undefined array key "id"
in APPPATH\Controllers\Crud.php on line 50.
 1 APPPATH\Controllers\Crud.php(50): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 50)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->actualizar()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:35:22 --> Undefined array key "id"
in APPPATH\Controllers\Crud.php on line 50.
 1 APPPATH\Controllers\Crud.php(50): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 50)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->actualizar()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 05:45:24 --> Undefined array key "id"
in APPPATH\Controllers\Crud.php on line 50.
 1 APPPATH\Controllers\Crud.php(50): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Crud.php', 50)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->actualizar()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 06:06:54 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 06:21:48 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 06:21:54 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 06:21:54 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 06:29:10 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 06:29:48 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 06:29:53 --> preg_replace(): No ending delimiter '/' found
in APPPATH\Controllers\Login.php on line 26.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'preg_replace(): No ending delimiter \'/\' found', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Login.php', 26)
 2 APPPATH\Controllers\Login.php(26): preg_replace('/[^a-zA-Z0-9.-]', '', '')
 3 SYSTEMPATH\CodeIgniter.php(934): App\controllers\Login->validarIngreso()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\controllers\Login))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 07:04:20 --> Undefined array key 0
in APPPATH\Views\actualizar.php on line 4.
 1 APPPATH\Views\actualizar.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\actualizar.php', 4)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\actualizar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('actualizar', [], true)
 5 APPPATH\Controllers\Crud.php(71): view('actualizar', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->obtenerNombre('8')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-05 07:04:51 --> Undefined array key 0
in APPPATH\Views\actualizar.php on line 4.
 1 APPPATH\Views\actualizar.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\actualizar.php', 4)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\actualizar.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('actualizar', [], true)
 5 APPPATH\Controllers\Crud.php(71): view('actualizar', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Crud->obtenerNombre('8')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Crud))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
